x-pack/heartbeat/monitors/browser/project.go

```diff
@@ -140,17 +160,19 @@ func (p *Project) jobs() []jobs.Job {
 
 	isScript := p.projectCfg.Source.Inline != nil
 	ctx := context.WithValue(p.ctx, synthexec.SynthexecTimeout, p.projectCfg.Timeout+30*time.Second)
+	sFields := p.StdFields()
 
 	if isScript {
 		src := p.projectCfg.Source.Inline.Script
-		j = synthexec.InlineJourneyJob(ctx, src, p.Params(), p.StdFields(), p.extraArgs()...)
+		j = synthexec.InlineJourneyJob(ctx, src, p.Params(), sFields, p.extraArgs(sFields.Origin != "")...)
```

Comment:
TIL: Does only project based monitors have this origin field set?

Reply:
Only UI monitors